lib/client/db/dbcmd/dbcmd.go

```diff
@@ -177,12 +179,28 @@ func (c *CLICommandBuilder) GetConnectCommand() (*exec.Cmd, error) {
 		return c.getSnowflakeCommand(), nil
 
 	case defaults.ProtocolElasticsearch:
-		return c.getElasticsearchCommand(), nil
+		return c.getElasticsearchCommand()
 	}
 
 	return nil, trace.BadParameter("unsupported database protocol: %v", c.db)
 }
 
+// GetConnectCommandOptions returns optional connection commands for protocols that offer multiple options.
+// Otherwise, it falls back to GetConnectCommand.
+// The keys in the returned map are command descriptions suitable for display to the end user.
+func (c *CLICommandBuilder) GetConnectCommandOptions() (map[string]*exec.Cmd, error) {
```

Comment:
Why we need to use map ? The order is not deterministic so it will mean that the output can be different for each `tsh` call. Can we just return an array with a wrapper for exec.Cmd and `name string` ? 

The `GetConnectCommandOptions` returns multiple command with a string so maybe `GetConnectCommands`  or `GetMultiConnectCommands` will be better naming.

Reply:
> Can we just return an array with a wrapper for exec.Cmd and name string ?

@smallinsky what would be the signature for this?

Reply:
What do you think about something like bolow where a map is replaced by an array. 
```
type Cmd struct {
    Name string
    Cmd *exec.Cmd
}

func GetMultiConnectCommands() ([]Cmd, error) {
}
```

Reply:
I think the benefit is minimal over `map[string]*exec.Cmd`, but I can go with this.